<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="arriba_f">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="medio_arriba_f">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <widget class="QLabel" name="titulo">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>50</y>
            <width>279</width>
            <height>68</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>URW Gothic</family>
            <pointsize>50</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>RoboTEA</string>
          </property>
         </widget>
         <widget class="QLabel" name="icono_robot">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>30</y>
            <width>96</width>
            <height>96</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="iconos/recursos.qrc">:/iconos/icons8-larva-del-moscardón-96.png</pixmap>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="medio_f">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="izq_cont">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="menu_cont">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <widget class="QLabel" name="nuevasesion_label">
             <property name="geometry">
              <rect>
               <x>250</x>
               <y>250</y>
               <width>117</width>
               <height>17</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Nueva Sesión</string>
             </property>
            </widget>
            <widget class="QPushButton" name="nuevaSesionButton">
             <property name="geometry">
              <rect>
               <x>210</x>
               <y>80</y>
               <width>162</width>
               <height>156</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="iconos/recursos.qrc">
               <normaloff>:/iconos/cozmo.png</normaloff>:/iconos/cozmo.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLabel" name="nuevasesion_label_2">
             <property name="geometry">
              <rect>
               <x>550</x>
               <y>250</y>
               <width>157</width>
               <height>17</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Ubuntu</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Editar ordenes Cozmo</string>
             </property>
            </widget>
            <widget class="QPushButton" name="editarOrdenesButton">
             <property name="geometry">
              <rect>
               <x>540</x>
               <y>80</y>
               <width>162</width>
               <height>156</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="iconos/recursos.qrc">
               <normaloff>:/iconos/editar_cozmo_150.png</normaloff>:/iconos/editar_cozmo_150.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="abaj_f">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="version">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="text">
           <string>Versión 1.</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="help">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>0</y>
            <width>44</width>
            <height>38</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="iconos/recursos.qrc">
            <normaloff>:/iconos/icons8-ayuda-64.png</normaloff>:/iconos/icons8-ayuda-64.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="iconos/recursos.qrc"/>
  <include location="iconos/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
